ynput/ayon-frontend
# Op-6285 make project attribute defaults editable outside anatomy

### Testing
1. Project managers dashboard
2. Click edit top right in project details
3. Change some stuff
4. Click save
5. Attrib should update

![image](https://github.com/ynput/ayon-frontend/assets/49156310/cb226bbd-fc22-43bc-aa9c-e1828e5a8bb7)
